src/legacy/status_im/bottom_sheet/view.cljs

```diff
@@ -207,7 +207,7 @@
                            [@show-bottom-sheet? @expanded? @gesture-running?])
 
          [:<>
-          [rn/touchable-without-feedback {:on-press (when backdrop-dismiss? close-bottom-sheet)}
+          [rn/pressable {:on-press (when backdrop-dismiss? close-bottom-sheet)}
```

Comment:
Please double check this.

I noticed some styles were breaking when I tried to remove all the `touchable-without-feedback` wrapping a `reanimated/view`

Reply:
yes thanks, `touchable-without-feedback` doesn't add a container, so if the child uses absolute positioning or flex  it will work differently with pressable it's better not to use `touchable-without-feedback` at all

Reply:
I agree I had to deal with weird crashes in rn-73 upgrade due to `touchable-without-feedback` -> https://github.com/status-im/status-mobile/pull/18563#issuecomment-1911511881
Its better if we use `rn/pressable` everywhere